modules/creating-config-files-cluster-install-oci.adoc

```diff
@@ -17,11 +17,17 @@ You can also use the Agent-based Installer to generate or accept Zero Touch Prov
 * You read the documentation on xref:../../installing/installing-preparing.html#installing-preparing-selecting-cluster-type[Selecting a cluster installation method and preparing it for users].
 * You have read the xref:../../installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.html#about-the-agent-based-installer[Preparing to install with the Agent-based Installer] documentation.
 * You downloaded the xref:../../installing/installing_with_agent_based_installer/installing-with-agent-based-installer.html#installing-ocp-agent-retrieve_installing-with-agent-based-installer[Agent-Based Installer] and the command-line interface (CLI) from Red Hat’s Hybrid Cloud Console.
-* For a disconnected environment, you created a container image registry, such as Red Hat Quay. See xref:../../installing/disconnected_install/installing-mirroring-creating-registry.html#mirror-registry-introduction_installing-mirroring-creating-registry[Mirror registry for Red Hat OpenShift introduction].
 * You have logged into the {product-title} with administrator privileges. 
 
 .Procedure
 
+. For a disconnected environment, you mirrored the {product-mirror-registry} to your local container image registry, such as {quay}. See xref:../../installing/disconnected_install/installing-mirroring-installation-images.html#installation-mirror-repository_installing-mirroring-installation-images[Mirroring the {product-title} image repository].
```

Comment:
Are the [mirror registry for Red Hat OpenShift](https://docs.openshift.com/container-platform/4.15/installing/disconnected_install/installing-mirroring-creating-registry.html) and the [OpenShift Container Platform image repository](https://docs.openshift.com/container-platform/4.15/installing/disconnected_install/installing-mirroring-installation-images.html#installation-mirror-repository_installing-mirroring-installation-images) the same thing?

Reply:
Do I need to document how a customer can establish a local container image registry?

Reply:
Hi @pawanpinjarkar , @rwsu , and @bfournie . WOuld you be OK to review these comments and let me know either link is suitable to explain how to create local container image registry?

I think this is the last item to address for the ABI work on this PR? Let me know if anything else stands out.

Reply:
@bfournie please guide. I think [mirror registry for Red Hat OpenShift](https://docs.openshift.com/container-platform/4.15/installing/disconnected_install/installing-mirroring-creating-registry.html) should be ok as it has a ref inside for [OpenShift Container Platform image repository](https://docs.openshift.com/container-platform/4.15/installing/disconnected_install/installing-mirroring-installation-images.html#installation-mirror-repository_installing-mirroring-installation-images)

Reply:
Agree that the mirror registry link at 
https://docs.openshift.com/container-platform/4.14/installing/disconnected_install/installing-mirroring-installation-images.html is fine to reference to describe how mirroring should be done